### PR TITLE
Adding skipConfirmation to to 2to3 move config

### LINKED ISSUE #158: Is there a way to bypass 2to3 confirm ?
we are using the helm 2to3 convert grammatically in c# ,is there a way to bypass this confirmation : 
> helm3 2to3 move config
2020/07/29 10:50:21 WARNING: Helm v3 configuration may be overwritten during this operation.
2020/07/29 10:50:21
[Move Config/confirm] Are you sure you w**ant to move the v2 configuration? [y/N]:**

we are using the c# start process : 
 new Process
                {
                    StartInfo = new ProcessStartInfo
                    {
                        FileName = "helm3",
                        Arguments = $"{arguments}",
                        RedirectStandardOutput = true,
                        RedirectStandardError = true,
                        CreateNoWindow = true,
                        UseShellExecute = false
                    }
                };
what will be the best way to send yes ,or to bypass this message ?
thank you.

### PR DESCRIPTION
**Pull request by digeler**
Add a flag to bypass confirmation when using 2to3 move config 

Close #158

### CODE REVIEW

File: cmd/environment.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ func New() *EnvSettings {
 // AddBaseFlags binds base flags to the given flagset.
 func (s *EnvSettings) AddBaseFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.DryRun, "dry-run", false, "simulate a command")
+
```
Comment (marckhouzam) [inline cmd/environment.go:39]: I guess this file does not need to change?

File: cmd/move_config.go [hunk 2]
```diff
@@ -66,12 +71,18 @@ func Move(dryRun bool) error {
...
 	log.Println()
-	doCleanup, err := utils.AskConfirmation("Move Config", "move the v2 configuration")
+	if skipConfirmation {
+		log.Println("Skipping confirmation before performing move.")
+		doCleanup = true
+		err = nil
+	} else {
+		doCleanup, err = utils.AskConfirmation("Move Config", "move the v2 configuration")
+	}
 	if err != nil {
```
Comment (marckhouzam) [inline cmd/move_config.go:81]: I think you can move this `err` check inside the above `else` clause. That way you don't need to declare `err` globally anymore.

File: cmd/move_config.go [hunk 3]
```diff
@@ -40,7 +40,8 @@ func newMoveConfigCmd(out io.Writer) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	settings.AddBaseFlags(flags)
+	settings.AddFlags(flags)
```
Comment (marckhouzam) [inline cmd/move_config.go:43]: I don't think this command should have all the flags as it doesn't communicate with the kubernetes cluster.

Did you mean to change this line?

**Comment by marckhouzam** [comment]
Also @digeler you will need to sign the DCO.  To do that you'll need sign each commit.  I suggest you squash your commits into a single one and sign that one then do a force-push.

File: cmd/environment.go (hunk 1 above)
Comment (digeler) [reply cmd/environment.go:39]: correct removed blank lines

File: cmd/move_config.go (hunk 2 above)
Comment (digeler) [reply cmd/move_config.go:81]: fixed

File: cmd/environment.go (hunk 1 above)
Comment (digeler) [reply cmd/environment.go:39]: fixed

**Comment by digeler** [comment]
why am i still getting this : 
DCO
You only have one commit incorrectly signed off! To fix, first ensure you have a local copy of your branch by checking out the pull request locally via command line. Next, head to your local branch and run:

git commit --amend --signoff
Now your commits will have your sign off. Next run

git push --force-with-lease origin master
Commit sha: 0618587, Author: Dinor Geler, Committer: Dinor Geler; The sign-off is missing.

**Comment by marckhouzam** [comment]
If you look at the list of commits of this PR you'll see there are too many.  You should bring it down to one with your changes only.
https://github.com/helm/helm-2to3/pull/159/commits

**Comment by digeler** [comment]
am closing this one ill create a new one ,sorry for the mess

**Pull request closed without merging**